docs/user/metrics/timespan.md

```diff
@@ -78,8 +93,10 @@ assertTrue(Auth.loginTime.testGetValue() > 0)
 ## Recorded errors
 
 * `invalid_value`: If recording a negative timespan.
```

Comment:
nit: could this be a two layers list?

- `invalid_value`:
  - If starting a timer..
  - If stopping a timer...